### ISSUE TITLE
Using for other language?

### ISSUE THREAD

**Issue by SuPanther** [ajitrajasekharan/unsupervised_NER#2]
Can I achieve this unsupervised method of NER using multi BERT for other languages?

**Comment by ajitrajasekharan** [comment]
Yes, it should be possible. We just need a model training on the language. Ideally pretrained with a custom vocab on a corpus that has entities of interest to us but not required.

Ajit
